test/mammoth.tests.js

```diff
@@ -197,7 +197,7 @@ describe('mammoth', function() {
         var buffer = fs.readFileSync(docxPath);
         return mammoth.convertToHtml({buffer: buffer}).then(function(result) {
             assert.equal(result.value, '');
-            assert.deepEqual(result.messages, [results.warning("could not find external image 'tiny-picture.png', path of input document is unknown")]);
+            assert.equal(result.messages[0].message, results.error("could not find external image 'tiny-picture.png', path of input document is unknown").message);
```

Comment:
This weakens the assertion in that we no longer check the severity of the message (which is really the point of the exercise), nor do we check that there's exactly one message. Also, I don't think there's a need to build the expected error, only to immediately pull the message out: you may as well just use the expected message directly.